style.css

```diff
@@ -1,11 +1,11 @@
 * {
   margin: 0;
   padding: 0;
+  box-sizing: border-box;
   background-repeat: no-repeat;
   text-decoration: none;
   list-style: none;
   font-family: 'Roboto Slab', serif;
```

Comment:
- [ ] [REQUIRED] As mentioned by the previous reviewer, that the font family, font-size does not match that of the design. I think the issue is that you are not importing the font into your CSS page file. I think if you use `@import url('https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap');` at the top of your CSS file, it will solve the problem. Please the import statement for other fonts to your CSS. 

The image below shows how the `font-family: inter` is not affecting your element text as expected.

![image](https://user-images.githubusercontent.com/84156714/186996374-c34c0191-c560-4eb2-9cc3-142a6923698e.png)